### PR TITLE
Better history table

### PR DESCRIPTION
**Pull request by DrAnax**
Initial effort

### CODE REVIEW

File: server/src/http_shared_missing_scores.go [hunk 1]
```diff
@@ -65,13 +65,14 @@ func httpSharedMissingScores(c *gin.Context) {
 		}
 	}
 
+	lastIndex := len(playerNames) - 1
 	data := &TemplateData{ // nolint: exhaustivestruct
 		Title:               "Missing Scores",
-		NamesTitle:          "Missing Scores for [" + strings.Join(playerNames, ", ") + "]",
+		NamesTitle:          "Missing Scores for " + strings.Join(playerNames[:lastIndex], ", ") + " and " + playerNames[lastIndex],
```
Comment (Zamiell) [inline server/src/http_shared_missing_scores.go:71]: isnt this bugged when there is only one player

File: server/src/http_history.go [hunk 2]
```diff
@@ -63,5 +63,5 @@ func httpHistory(c *gin.Context) {
 	if len(playerNames) > 1 {
 		data.NamesTitle = "Game History for [" + strings.Join(playerNames, ", ") + "]"
 	}
-	httpServeTemplate(w, data, "profile", "history")
+	httpServeTemplate(w, data, "profile-new", "history")
```
Comment (Zamiell) [inline server/src/http_history.go:66]: why are we using profile-new instead of profile

Comment (DrAnax) [reply server/src/http_history.go:66]: I was going back and forth between profile and profile-new. In the end I managed to make all the pages as I liked, but I forgot to transfer back.

File: server/src/http_shared_missing_scores.go (hunk 1 above)
Comment (DrAnax) [reply server/src/http_shared_missing_scores.go:71]: image

**Comment by Zamiell** [comment]
you need to rebase this on main because i am seeing unrelated changes to this PR

File: server/src/http_shared_missing_scores.go (hunk 1 above)
Comment (Zamiell) [reply server/src/http_shared_missing_scores.go:71]: ok, nevermind then

**Pull request closed without merging**